.github/workflows/quicktest.yml

```diff
@@ -0,0 +1,79 @@
+name: Quick Tests
+
+on:
+  push:
+    branches-ignore:
+      - master
+      - develop
+
+jobs:
+  # Performs some quick tests.
+  # This is a much quicker test suite which only runs the unit tests and linting
+  # against the low/high supported PHP/PHPCS combinations.
+  #
+  # Performs the following steps:
+  # - Checks out the repository.
+  # - Configures caching for Composer.
+  # - Sets up PHP.
+  # - Installs PHPCS.
+  # - Install Composer dependencies.
+  # - Runs the Composer linting script (if desired).
+  # - Runs Composer run-tests script.
+  quick-tests:
+    name: PHP ${{ matrix.php }} on PHPCS ${{ matrix.phpcs_branch }}${{ matrix.lint && ' with linting' || '' }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: [ '7.4', '5.4' ]
```

Comment:
Pro-tip: `7.4` can actually probably better be changed to `latest` for this job. `latest` will always translate to the last stable release of PHP.

Reply:
Updated! I had left it as 7.4 as that is what the TravisCI file had and I was aiming for a apples to apples conversion. But this comment documents that, so it now uses `latest`. I usually default to using an explicit version to prevent unintended changes, but `latest` works for me if that's the preference.

Reply:
Looks like using PHP 8.0 was causing the [workflow to fail](https://github.com/WordPress/WordPress-Coding-Standards/runs/1597986278?check_suite_focus=true). I've reverted it for now in the interest of getting the automated tests fixed and we can revisit this after.